Integrations/FortiGate/FortiGate.py

```diff
@@ -429,6 +470,86 @@ def create_firewall_service_request(service_name, tcp_range, udp_range):
     return response
 
 
+def ban_ip(ip_addresses_array, time_to_expire=0):
+    uri_suffix = 'monitor/user/banned/add_users/'
+
+    payload = {
+        'ip_addresses': ip_addresses_array,
+        'expiry': time_to_expire
+    }
+
+    response = http_request('POST', uri_suffix, data=json.dumps(payload))
+    return response
+
+
+def ban_ip_command():
+    ip_addresses_string = demisto.args()['ip_address']
+    ip_addresses_array = argToList(ip_addresses_string)
+    for ip_address in ip_addresses_array:
+        if not is_ip_valid(ip_address, True):
```

Comment:
The True seems off without it 
```suggestion
        if not is_ip_valid(ip_address, accept_v6_ips=True):
```